modules/apps/commerce/commerce-price-list-web/src/main/resources/META-INF/resources/edit_instance_tier_price_entry.jsp

```diff
@@ -32,16 +32,9 @@ CommercePriceList commercePriceList = commercePriceEntry.getCommercePriceList();
 CommerceCurrency commerceCurrency = commercePriceList.getCommerceCurrency();
 
 BigDecimal price = BigDecimal.ZERO;
-BigDecimal promoPrice = BigDecimal.ZERO;
 
-if ((commercePriceEntry != null) && (commerceTierPriceEntry != null)) {
-	if (commerceTierPriceEntry.getPrice() != null) {
-		price = commerceCurrency.round(commerceTierPriceEntry.getPrice());
-	}
-
-	if (commerceTierPriceEntry.getPromoPrice() != null) {
-		promoPrice = commerceCurrency.round(commerceTierPriceEntry.getPromoPrice());
-	}
+if ((commercePriceEntry != null) && (commerceTierPriceEntry != null) && (commerceTierPriceEntry.getPrice() != null)) {
```

Comment:
@riccardo-alberti is commercePriceEntry != null  condiction needed?

Reply:
@marco-leo i removed it as the price is taken from tierPriceEntry